src/_filter/collection/where.js

```diff
@@ -12,6 +12,9 @@ angular.module('a8m.where', [])
   .filter('where', function() {
     return function (collection, object) {
 
+      if(isUndefined(collection)) {
+        return false;
+      }
```

Comment:
Because the return is different in each case. So I thought better to separate them.